tests/test_api_tokens.py

```diff
@@ -1,50 +0,0 @@
-from flask import json
```

Comment:
Why are these tests removed?

Reply:
the same file is also in core folder, so this is just duplicate - https://github.com/superdesk/newsroom-core/blob/develop/tests/core/test_api_tokens.py